config.py

```diff
@@ -1,7 +1,8 @@
 from glue.config import menubar_plugin
-from statswidget import StatsGui
+from statwidget import StatsGui
 
 @menubar_plugin("Show stats")
 def my_plugin(session, data_collection):
     ex = StatsGui(data_collection)
     ex.show()
+    session.ex = ex
```

Comment:
This is a hack to make sure the dialog stays open - otherwise the widget is closed when the function finishes.